sound/soc/sof/trace.c

```diff
@@ -291,6 +314,8 @@ void snd_sof_release_trace(struct snd_sof_dev *sdev)
 			"error: fail in snd_sof_dma_trace_release %d\n", ret);
 
 	sdev->dtrace_is_enabled = false;
+	sdev->dtrace_eof = true;
```

Comment:
.... here. dtrace_is_enabled is force to false at the beginning of the function, then set here. No such assignment for dtrace_eof so not sure if this init is needed.

Reply:
@plbossart This is correct. Here FW tracing is disabled -> we want to give user-space a chance to read the data out before resume kicks in (-> trace training in latest patchset).